VideoLocker/src/main/java/org/edx/mobile/view/MyCoursesListActivity.java

```diff
@@ -32,10 +42,39 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         configureDrawer();
         setTitle(getString(R.string.label_my_courses));
+        /* Add a CoordinatorLayout so that the Snackbar can be dismissed. It's added as a
+         * new content view for convenience, to avoid messing with the existing layout. As
+         * we implement Snackbar notifications globally throughout the app, we should
+         * probably set up the CoordinatorLayout as a global content view parent in
+         * BaseFragmentActivity.
+         */
+        coordinatorLayout = new CoordinatorLayout(this);
```

Comment:
Maybe add a TODO here to remove this logic once we switch to using `CoordinatorLayout` all over the app.